.bumpversion.cfg

```diff
@@ -1,16 +1,30 @@
 [bumpversion]
-current_version = 0.7.2
+current_version = 0.7.6_dev0
```

Comment:
I like that you reincarnated bumpversion, I also forgot about it in the meantime...